app/handlers/login.js

```diff
@@ -1,37 +1,43 @@
-const passport = require('passport');
-const LocalStrategy = require('passport-local').Strategy;
-const mysql = require(__base+ '/app/modules/common/mysql');
-const bcrypt = require('bcryptjs');
-const signupModule = require(__base + '/app/modules/registration/signup');
+const passport = require("passport");
```

Comment:
why is all single quote (') replaced by double quotes (")
this should not happen- usually we use single as it is easy to write.

Reply:
I think it was the prettier plugin will take keep in mind from now on.